app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -742,6 +745,29 @@ public function hitPage($page, $request, $code = '200')
             $hit->setBrowserLanguages($languages);
         }
 
+        //device granularity
+        $dd = new DeviceDetector($request->server->get('HTTP_USER_AGENT'));
+
+        $dd->parse();
+
+        $deviceStat = new StatDevice();
+
+        $deviceStat->setChannel('Hit');
```

Comment:
Thoughts on this: Should we use model names for channel? That way we can simply do getModel($channel)->getEntity($channelId) without having to know what the entity actually is. I know we don't have a hit model yet but using something like `page.hit` here would allow us to  add support for a HitModel later in PageBundle. Same for where setChannel('Email'); using setChannel('email') will allow us to do getModel('email')->getEntity($channelId);